### PR TITLE
show held item shows icon beside now

### PR DESCRIPTION
**Pull request by alsoandanswer**
yep

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

File: code/game/objects/items.dm [hunk 1]
```diff
@@ -670,7 +670,7 @@ var/list/global/slot_flags_enumeration = list(
 /obj/item/proc/showoff(mob/user)
 	for (var/mob/M in view(user))
 		if(!user.is_invisible_to(M))
-			M.show_message("<b>[user]</b> holds up [src]. <a HREF=?src=\ref[M];lookitem=\ref[src]>Take a closer look.</a>",1)
+			M.show_message("<b>[user]</b> holds up [icon2html(src)] [src]. <a HREF=?src=\ref[M];lookitem=\ref[src]>Take a closer look.</a>",1)
```
Comment (NonQueueingMatt) [inline code/game/objects/items.dm:673]: This is not proper usage of icon2html. Did you test this? It won't work.

Icon2html needs a second argument, the list of people that would receive the icon. Here that would be `viewers(user)` or similiar.

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

**Comment by alsoandanswer** [comment]
dreamseeker_LNhbxW5cTV

**Comment by alsoandanswer** [comment]
Fixed it

**Review by NonQueueingMatt — APPROVED** [review]

**Review by Doxxmedearly — APPROVED** [review]

**Pull request merged**